pkg/arvo/sys/zuse.hoon

```diff
@@ -6311,7 +6311,7 @@
     |_  ent/_`(map term @t)`[[%apos '\''] ~ ~]
     ::                                                  ::  ++apex:de-xml:html
     ++  apex                                            ::  top level
-      =+  spa=;~(pose comt whit)
+      =+  spa=;~(pose comt whit decl)
       %+  knee  *manx  |.  ~+
       %+  ifix  [(star spa) (star spa)]
```

Comment:
The XML spec requires declarations appear only as the very first element in a document. Adding it to the "spacing" parser, allowing it to occur multiple times, and even at the end of the document, doesn't seem entirely correct to me.

Considering doing `;~(plug (punt decl) (star spa))` as the prefix parser you give to `ifix`.